### ISSUE TITLE
Accessibility Page - Checklist Design

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#2249]
Amanda and I connected and discussed if the designers could help figure out if there's another way that that is easier to view the context within the checklist she created. [Currently, the user needs to open the accordion like list items to view the content within each row.](https://deploy-preview-2202--keen-mayer-a86c8b.netlify.app/foundation/wcag-checklist#global-code)

Screenshots:
<img width="873" alt="Screen Shot 2022-01-24 at 10 50 13 AM" src="https://user-images.githubusercontent.com/62117454/150845982-8fbeaa7d-ae3f-4b2b-86f4-e9a068f5e305.png">
<img width="856" alt="Screen Shot 2022-01-24 at 11 01 04 AM" src="https://user-images.githubusercontent.com/62117454/150846879-88109b1c-811a-467e-830d-1edd09ae0a66.png">

To do's:

- [x] Research a potential other way of visually showing the checklist
- [x] Make sure it is accessible for both keyboard and mouse users
- [x] Create a visual example

**Comment by vavalos5** [comment]
@amandadupell I was able to do some noodling on how we might be able to better visualize this. I think what you've got is great already. Let's continue to have the list layout you've got. Here's what i'm thinking:

- Remove the ascending and descending buttons to take the accordion like behavior away. 
- Extend the container in which the text lives in to fill more of the page.
- Make heading `small-bold` we could do `default-bold` too, might look better from a hierarchal view.

<img width="798" alt="Screen Shot 2022-01-27 at 4 25 27 PM" src="https://user-images.githubusercontent.com/62117454/151466400-339c06fd-4d4b-4449-a668-e5ac1691d8f1.png">

I've included a screenshot above ^and the [link of where I created this example](https://www.figma.com/file/xKOmIiwc6h3yPsdLERxCbV/Accessibility?node-id=549%3A236). Though, the checkboxes aren't correctly placed. Let me know if you've got any questions or your thoughts.
